src/platforms/launchpad-target/platform.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * This file is part of the Black Magic Debug project.
+ *
```

Comment:
Please add your name and contact email  in the copyright.

That means you do at least for now some long time commitment to care for this platform

Reply:
Done. Hope I can find enough time in the future, though.